cmake/SetupPackages.cmake

```diff
@@ -94,8 +94,6 @@ if (RAJA_ENABLE_TESTS)
       CMAKE_ARGS                
           -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
           -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
-          -DCMAKE_C_FLAGS=${CMAKE_C_FLAGS}
```

Comment:
Poking at that might be a good idea.  I had something in there to ignore some of that, but I'm not sure I got it all.